swupd/swupd.go

```diff
@@ -539,6 +539,9 @@ func setMirror(swupdArgs []string, t string) (string, error) {
 // SetHostMirror executes the "swupd mirror" to set the Host's mirror
 func SetHostMirror(url string, allowInsecureHTTP bool) (string, error) {
 	if urlErr := network.CheckURL(url); urlErr != nil {
+		if strings.Contains(urlErr.Error(), "60") {
+			return "", fmt.Errorf(utils.Locale.Get("SSL certificate problem"))
```

Comment:
Use `errors.Errorf` to enable stack trace.

Reply:
In this instance, we do not want the stack trace as this string is display to the user in the UI.